Signal/src/call/CallService.swift

```diff
@@ -117,35 +117,67 @@ private class SignalCallData: NSObject {
     let rejectReadyToSendIceUpdatesPromise: ((Error) -> Void)
     let readyToSendIceUpdatesPromise: Promise<Void>
 
-    weak var localVideoTrack: RTCVideoTrack? {
-        didSet {
+    private weak var localVideoTrack_: RTCVideoTrack?
+    var localVideoTrack: RTCVideoTrack? {
```

Comment:
I modified `SignalCallData`'s accessors to more strictly enforce that we only access these properties on the main thread.